platform/darwin/src/MGLFillStyleLayer.mm

```diff
@@ -0,0 +1,25 @@
+#import "MGLFillStyleLayer.h"
+
+#include "MGLStyleLayer_Private.hpp"
```

Comment:
So far, we’ve named Objective-C++ headers with .h extensions, because .hpp implies C++. I can see how using .hpp would be useful, to avoid accidentally making C++-infected code part of our public API. But that should be a separate change applied consistently to the whole codebase.